pydropsonde/circles.py

```diff
@@ -219,6 +219,44 @@ def interpolate_na_sondes(self, method="cubic", max_gap=1500, thresh=4):
 
         return self
 
+    def remove_values(self, n_gap=3, keep_sfc=1000):
```

Comment:
This function name is not very telling. From reading the code, I could assume that something like:
```suggestion
    def create_regular_gaps_in_data_values(self, n_gap=3, keep_sfc=1000):
```
could be more fitting.